docs/index.md

```diff
@@ -23,6 +41,12 @@ of your platform.
   source hardware and open instruction set architecture. Feel free to check the
```

Comment:
If we in all documents use the term `open-source`, I think that we should also change the `open source` in line 40.

Reply:
I went through all the documentation. All occurrences of the phrase `open-source` (excluding proper names) has been fixed. I have no further comments.